mesh/mesh.go

```diff
@@ -376,20 +377,21 @@ func (m *Mesh) AccumulateRewards(rewardLayer types.LayerID, params Config) {
 		return
 	}
 
-	ids := make(map[string]struct{})
-	uq := make(map[string]struct{})
+	var ids []string
```

Comment:
if you know the size of `l.Blocks` you can initialize it and insert them by index